modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsCompletionVelocity.testcase

```diff
@@ -75,12 +75,66 @@ definition {
 	}
 
 	@description = "LPS-97665 - Verify that an item out of the timeframe selected is not considered in the velocity result"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test ItemOutOfTimeFrameIsNotConsideredInVelocityResult {
+		property portal.acceptance = "true";
+
+		WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
+
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Process Builder");
+
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			panel = "Site Administration",
+			portlet = "Blogs");
+
+		Blogs.addWithWorkflow(
+			entryContent = "Test Content",
```

Comment:
Could you please change this macro to 
`JSONBlog.addEntry( entryContent = "Blogs Entry Content", entryTitle = "Blogs Entry Title"); `